FAQ.md

```diff
@@ -0,0 +1,29 @@
+# Here we have listed the most common questions, so you can quickly resolve any issues.
+
+**Node is not installed on my computer. Where can I download it from?**
+--
+Head over to https://nodejs.org/en/ and download the latest LTS version for your operating system.
+
+**What code editor should I use?**
+--
+Generally, we recommend Atom (https://atom.io) or VS Code (https://code.visualstudio.com) for development. Both are open-source and have many plugins with support for many languages.
+
+**Where can I look up docs for JavaScript?**
+--
+MDN (https://developer.mozilla.org/en-US/) and w3schools (https://www.w3schools.com) have everything you need for web development.
+
+**How do I install an npm package?**
+--
+NPM packages can be looked up at https://www.npmjs.com, and install from your command line emulator of choice using `npm install <packagename>`. Use the optional `--save` flag to save the dependency to you package's `package.json` file.
```

Comment:
As of npm 5.0.0 the `--save` is not required, since the installed packages are automatically added to the `package.json` file. Usage of the `-D` or `-G` for developer and global package installation could be mentioned.